time    tick      # Time
float64 px		  # Navigation-frame Position X (m)
float64 py		  # Navigation-frame Position Y (m)
float64 pz		  # Navigation-frame Position Z (m)
float64 rx		  # Roll (rad)
float64 ry		  # Pitch (rad)
float64 rz		  # Bearing (rad)
float64 vx		  # Navigation-frame Position X (m/s)
float64 vy		  # Navigation-frame Position Y (m/s)
float64 vz		  # Navigation-frame Position Z (m/s)
float64 ax		  # Pitch (rad/s)
float64 ay		  # Roll (rad/s)
float64 az		  # Yaw (rad/s)
float64 thrust	  # Thrust force
float64 battery	  # Battery remaining (proportion)